{% extends 'main/base.html' %}
{% load bootstrap4 %}

{% block title %}{% endblock %}

{% block content %}
<div class="container">
        <div class="row">
            <div class="col">
        <h1>Annotation Documentations</h1>
        
        <dl class="row">
          <dt class="col-sm-3">1. splicing</dt>
          <dd class="col-sm-8">splicing-change prediction for splicing consensus SNPs<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">splicing_consensus_ada_score</dd><dd class="col-sm-6">"splicing-change prediction for splicing consensus SNPs based on adaboost. If the score &gt;0.6, it predicts that the splicing will be changed, otherwise it predicts the splicing will not be changed."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">splicing_consensus_rf_score</dd><dd class="col-sm-6">splicing-change prediction for splicing consensus SNPs based on random forest.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">2. 1000Gp3</dt>
          <dd class="col-sm-8">Whole genome allele frequencies from the 1000 Genomes Project phase 3 data<p class="text-secondary">Allele frequencies (SNVs and indels)</p></dd>
          <dd class="col-sm-1"><a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">1000Gp3_AC</dd><dd class="col-sm-6">Alternative allele counts in the whole 1000 genomes phase 3 (1000Gp3) data.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">1000Gp3_AF</dd><dd class="col-sm-6">Alternative allele frequency in the whole 1000Gp3 data.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">1000Gp3_AFR_AC</dd><dd class="col-sm-6">Alternative allele counts in the 1000Gp3 African descendent samples.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">1000Gp3_AFR_AF</dd><dd class="col-sm-6">Alternative allele frequency in the 1000Gp3 African descendent samples.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">1000Gp3_AMR_AC</dd><dd class="col-sm-6">Alternative allele counts in the 1000Gp3 American descendent samples.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">1000Gp3_AMR_AF</dd><dd class="col-sm-6">Alternative allele frequency in the 1000Gp3 American descendent samples.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">1000Gp3_EAS_AC</dd><dd class="col-sm-6">Alternative allele counts in the 1000Gp3 East Asian descendent samples.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">1000Gp3_EAS_AF</dd><dd class="col-sm-6">Alternative allele frequency in the 1000Gp3 East Asian descendent samples.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">1000Gp3_EUR_AC</dd><dd class="col-sm-6">Alternative allele counts in the 1000Gp3 European descendent samples.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">1000Gp3_EUR_AF</dd><dd class="col-sm-6">Alternative allele frequency in the 1000Gp3 European descendent samples.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">1000Gp3_SAS_AC</dd><dd class="col-sm-6">Alternative allele counts in the 1000Gp3 South Asian descendent samples.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">1000Gp3_SAS_AF</dd><dd class="col-sm-6">Alternative allele frequency in the 1000Gp3 South Asian descendent samples.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">3. UK10K</dt>
          <dd class="col-sm-8">Total allele count in called genotypes in UK10K TWINSUK cohort.<p class="text-secondary">Allele frequencies (SNVs and indels)</p></dd>
          <dd class="col-sm-1"><a href="https://www.uk10k.org/studies/cohorts.html"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AC</dd><dd class="col-sm-6">Alternative allele count in called genotypes in UK10K cohorts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AF</dd><dd class="col-sm-6">Alternative allele frequency in called genotypes in UK10K cohorts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AN</dd><dd class="col-sm-6">Total allele count in called genotypes in UK10K cohorts.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">4. sensitive</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AC</dd><dd class="col-sm-6">Alternative allele count in called genotypes in UK10K cohorts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AF</dd><dd class="col-sm-6">Alternative allele frequency in called genotypes in UK10K cohorts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AN</dd><dd class="col-sm-6">Total allele count in called genotypes in UK10K cohorts.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">5. UTR3_miRNA_target</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AC</dd><dd class="col-sm-6">Alternative allele count in called genotypes in UK10K cohorts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AF</dd><dd class="col-sm-6">Alternative allele frequency in called genotypes in UK10K cohorts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AN</dd><dd class="col-sm-6">Total allele count in called genotypes in UK10K cohorts.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">6. aaref</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AC</dd><dd class="col-sm-6">Alternative allele count in called genotypes in UK10K cohorts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AF</dd><dd class="col-sm-6">Alternative allele frequency in called genotypes in UK10K cohorts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AN</dd><dd class="col-sm-6">Total allele count in called genotypes in UK10K cohorts.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">7. genename</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AC</dd><dd class="col-sm-6">Alternative allele count in called genotypes in UK10K cohorts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AF</dd><dd class="col-sm-6">Alternative allele frequency in called genotypes in UK10K cohorts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">UK10K_AN</dd><dd class="col-sm-6">Total allele count in called genotypes in UK10K cohorts.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">8. gnomAD</dt>
          <dd class="col-sm-8">"Exome and whole genome allele frequencies from the Genome Aggregation Database (123,136 exomes and 15,496 whole genomes)"<p class="text-secondary">Allele frequencies (SNVs and indels)</p></dd>
          <dd class="col-sm-1"><a href="http://gnomad.broadinstitute.org/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_AC</dd><dd class="col-sm-6">"Alternative allele count in the whole gnomAD exome samples (123,136 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_AF</dd><dd class="col-sm-6">"Alternative allele frequency in the whole gnomAD exome samples (123,136 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_AFR_AC</dd><dd class="col-sm-6">"Alternative allele count in the African/African American gnomAD exome samples (7,652 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_AFR_AF</dd><dd class="col-sm-6">"Alternative allele frequency in the African/African American gnomAD exome samples (7,652 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_AFR_AN</dd><dd class="col-sm-6">"Total allele count in the African/African American gnomAD exome samples (7,652 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_AMR_AC</dd><dd class="col-sm-6">"Alternative allele count in the Latino gnomAD exome samples (16,791 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_AMR_AF</dd><dd class="col-sm-6">"Alternative allele frequency in the Latino gnomAD exome samples (16,791 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_AMR_AN</dd><dd class="col-sm-6">"Total allele count in the Latino gnomAD exome samples (16,791 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_AN</dd><dd class="col-sm-6">"Total allele count in the whole gnomAD exome samples (123,136 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_ASJ_AC</dd><dd class="col-sm-6">"Alternative allele count in the Ashkenazi Jewish gnomAD exome samples (4,925 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_ASJ_AF</dd><dd class="col-sm-6">"Alternative allele frequency in the Ashkenazi Jewish gnomAD exome samples (4,925 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_ASJ_AN</dd><dd class="col-sm-6">"Total allele count in the Ashkenazi Jewish gnomAD exome samples (4,925 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_EAS_AC</dd><dd class="col-sm-6">"Alternative allele count in the East Asian gnomAD exome samples (8,624 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_EAS_AF</dd><dd class="col-sm-6">"Alternative allele frequency in the East Asian gnomAD exome samples (8,624 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_EAS_AN</dd><dd class="col-sm-6">"Total allele count in the East Asian gnomAD exome samples (8,624 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_FIN_AC</dd><dd class="col-sm-6">"Alternative allele count in the Finnish gnomAD exome samples (11,150 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_FIN_AF</dd><dd class="col-sm-6">"Alternative allele frequency in the Finnish gnomAD exome samples (11,150 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_FIN_AN</dd><dd class="col-sm-6">"Total allele count in the Finnish gnomAD exome samples (11,150 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_NFE_AC</dd><dd class="col-sm-6">"Alternative allele count in the Non-Finnish European gnomAD exome samples (55,860 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_NFE_AF</dd><dd class="col-sm-6">"Alternative allele frequency in the Non-Finnish European gnomAD exome samples (55,860 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_NFE_AN</dd><dd class="col-sm-6">"Total allele count in the Non-Finnish European gnomAD exome samples (55,860 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_OTH_AC</dd><dd class="col-sm-6">"Alternative allele count in other gnomAD exome samples (2,743 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_OTH_AF</dd><dd class="col-sm-6">"Alternative allele frequency in other gnomAD exome samples (2,743 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_OTH_AN</dd><dd class="col-sm-6">"Total allele count in other gnomAD exome samples (2,743 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_SAS_AC</dd><dd class="col-sm-6">"Alternative allele count in the South Asian gnomAD exome samples (15,391 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_SAS_AF</dd><dd class="col-sm-6">"Alternative allele frequency in the South Asian gnomAD exome samples (15,391 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_SAS_AN</dd><dd class="col-sm-6">"Total allele count in the South Asian gnomAD exome samples (15,391 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_exomes_flag</dd><dd class="col-sm-6">"information from gnomAD exome data indicating whether the variant falling within low-complexity (lcr) or segmental duplication (segdup) regions. The flag can be either ""."" for high-quality PASS or not reported/polymorphic in gnomAD exomes, ""lcr"" for within lcr, ""segdup"" for within segdup, or ""segdup;lcr"" for with both lcr and segdup."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_AC</dd><dd class="col-sm-6">"Alternative allele count in the whole gnomAD genome samples (15,496 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_AF</dd><dd class="col-sm-6">"Alternative allele frequency in the whole gnomAD genome samples (15,496 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_AFR_AC</dd><dd class="col-sm-6">"Alternative allele count in the African/African American gnomAD genome samples (4,368 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_AFR_AF</dd><dd class="col-sm-6">"Same as above, but with genome samples instead of exome."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_AFR_AN</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_AMR_AC</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_AMR_AF</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_AMR_AN</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_AN</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_ASJ_AC</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_ASJ_AF</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_ASJ_AN</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_EAS_AC</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_EAS_AF</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_EAS_AN</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_FIN_AC</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_FIN_AF</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_FIN_AN</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_NFE_AC</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_NFE_AF</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_NFE_AN</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_OTH_AC</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_OTH_AF</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_OTH_AN</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">gnomAD_genomes_flag</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">9. snoRNA/miRNA</dt>
          <dd class="col-sm-8">snoRNA and miRNA in human genome collected in miRBase r21 and snoRNABase v3<p class="text-secondary">other annotation</p></dd>
          <dd class="col-sm-1"><a href="http://www.mirbase.org/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">sno_miRNA_name</dd><dd class="col-sm-6">the name of snoRNA or miRNA if the site is located within (from miRBase/snoRNABase)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">sno_miRNA_type</dd><dd class="col-sm-6">the type of snoRNA or miRNA (from miRBase/snoRNABase)</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">10. SnpEff</dt>
          <dd class="col-sm-8">Pre-computed SnpEff annotations for all alternative SNVs based on human reference genome hg19 (Ensembl and RefSeq)<p class="text-secondary">Pre-computed gene-model based annotation (recommended for large-scale annotation)</p></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_CDS_position/CDS_len</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_Distance_to_feature</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_Effect</dd><dd class="col-sm-6">SnpEff consequence with Ensembl as gene model.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_Effect_impact</dd><dd class="col-sm-6">consequence impacts predicted by SnpEff.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_Exon_or_intron_rank/total</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_Gene_ID</dd><dd class="col-sm-6">Ensembl gene IDs corresponding to transcripts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_Gene_name</dd><dd class="col-sm-6">Ensembl gene names corresponding to transcripts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_HGVSc</dd><dd class="col-sm-6">variant in format of HGVSc.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_HGVSp</dd><dd class="col-sm-6">variant in format of HGVSp.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_LOF/NMD</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_LOF/NMD_gene_ID</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_LOF/NMD_gene_name</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_LOF/NMD_num_transcripts_affected</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_LOF/NMD_percent_transcripts_affected</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_Protein_position/Protein_len</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_Sequence_feature</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_Sequence_feature_impact</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_TF_ID</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_TF_binding_effect</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_TF_name</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_Transcript_ID</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_Transcript_biotype</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_Warnings</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_cDNA_position/cDNA_len</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_ensembl_summary</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_CDS_position/CDS_len</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_Distance_to_feature</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_Effect</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_Effect_impact</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_Exon_or_intron_rank/total</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_Gene_ID</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_Gene_name</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_HGVSc</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_HGVSp</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_LOF/NMD</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_LOF/NMD_gene_ID</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_LOF/NMD_gene_name</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_LOF/NMD_num_transcripts_affected</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_LOF/NMD_percent_transcripts_affected</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_Protein_position/Protein_len</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_Sequence_feature</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_Sequence_feature_impact</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_TF_ID</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_TF_binding_effect</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_TF_name</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_Transcript_ID</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_Transcript_biotype</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_Warnings</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_cDNA_position/cDNA_len</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SnpEff_refseq_summary</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">11. SIFT</dt>
          <dd class="col-sm-8">dbNSFP annotation by SFIT (Sorting Intolerant from Tolerant). SIFT predicts whether an amino acid substitution affects protein function based on sequence homology and the physical properties of amino acids.<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"><a href="https://sift.bii.a-star.edu.sg/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SIFT_converted_rankscore</dd><dd class="col-sm-6">"SIFTori scores were first converted to SIFTnew=1-SIFTori, then ranked among all SIFTnew scores in dbNSFP. The rankscore is the ratio of the rank the SIFTnew score over the total number of SIFTnew scores in dbNSFP."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SIFT_pred</dd><dd class="col-sm-6">"If SIFTori is smaller than 0.05 (rankscore&gt;0.55) the corresponding NS is predicted as ""D(amaging)""; otherwise it is predicted as ""T(olerated)"". "</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SIFT_score</dd><dd class="col-sm-6">SIFT score (SIFTori).</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">12. GWAS Catalog</dt>
          <dd class="col-sm-8">DNA variants associated with human diseases/phenotypes discovered in GWAS studies<p class="text-secondary">disease-related variants (SNVs and indels)</p></dd>
          <dd class="col-sm-1"><a href="https://www.genome.gov/gwastudies/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GWAS_catalog_pubmedid</dd><dd class="col-sm-6">pubmedid of the paper describing the association</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GWAS_catalog_rs</dd><dd class="col-sm-6">rs number according to GWAS catalog</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GWAS_catalog_trait</dd><dd class="col-sm-6">associated trait according to GWAS catalog</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">13. CADD</dt>
          <dd class="col-sm-8">Combined Annotation Dependent Depletion -- A genome-wide deleteriousness prediction score for DNA variants based on 63 sequence features (only SNV annotations are in WGSA)<p class="text-secondary">Functional prediction scores for non-coding SNVs</p></dd>
          <dd class="col-sm-1"><a href="https://cadd.gs.washington.edu/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">CADD_phred</dd><dd class="col-sm-6">"CADD phred-like score, ranges 1-99, the larger the number the more likely damaging"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">CADD_raw</dd><dd class="col-sm-6">"CADD raw score, the larger the number the more likely damaging"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">CADD_raw_rankscore</dd><dd class="col-sm-6">the rank of the CADD_raw score among all CADD_raw scores in genome</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">14. M-CAP</dt>
          <dd class="col-sm-8">dbNSFP annotation by Mendelian Clinically Applicable Pathogenicity (M-CAP) Score<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"><a href="http://bejerano.stanford.edu/mcap/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">M-CAP_pred</dd><dd class="col-sm-6">"Prediction of M-CAP score based on the authors' recommendation, ""T(olerated)"" or ""D(amaging)"". The score cutoff between ""D"" and ""T"" is 0.025.\n""+"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">M-CAP_rankscore</dd><dd class="col-sm-6">M-CAP scores were ranked among all M-CAP scores in dbNSFP. The rankscore is the ratio of the rank of the score over the total number of M-CAP scores in dbNSFP.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">M-CAP_score</dd><dd class="col-sm-6">M-CAP score (details in DOI the score the more likely the SNP has damaging effect.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">15. GRASP</dt>
          <dd class="col-sm-8">"Genome-Wide Repository of Associations Between SNPs and Phenotypes -- DNA variants associated with human diseases/phenotypes discovered in GWAS studies, including eQTLs and other quantitative trait scans"<p class="text-secondary">disease-related variants (SNVs and indels)</p></dd>
          <dd class="col-sm-1"><a href="https://grasp.nhlbi.nih.gov/Overview.aspx"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GRASP_PMID</dd><dd class="col-sm-6">PMID number by GRASP</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GRASP_ancestry</dd><dd class="col-sm-6">population ancestry of the samples on which the association test was based</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GRASP_p-value</dd><dd class="col-sm-6">p-value of the association test based on the SNP</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GRASP_phenotype</dd><dd class="col-sm-6">phenotype the SNP associated with</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GRASP_platform</dd><dd class="col-sm-6">SNP platform on which the association test was based</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GRASP_rs</dd><dd class="col-sm-6">rs number by GRASP</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">16. alt</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GRASP_PMID</dd><dd class="col-sm-6">PMID number by GRASP</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GRASP_ancestry</dd><dd class="col-sm-6">population ancestry of the samples on which the association test was based</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GRASP_p-value</dd><dd class="col-sm-6">p-value of the association test based on the SNP</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GRASP_phenotype</dd><dd class="col-sm-6">phenotype the SNP associated with</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GRASP_platform</dd><dd class="col-sm-6">SNP platform on which the association test was based</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GRASP_rs</dd><dd class="col-sm-6">rs number by GRASP</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">17. PROVEAN</dt>
          <dd class="col-sm-8">dbNSFP annotation by PROVEAN (Protein Variation Effect Analyzer)<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"><a href="http://provean.jcvi.org/index.php"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">PROVEAN_converted_rankscore</dd><dd class="col-sm-6">"PROVEANori were first converted to PROVEANnew=1-(PROVEANori+14)/28, then ranked among all PROVEANnew scores in dbNSFP. The rankscore is the ratio of the rank the PROVEANnew score over the total number of PROVEANnew scores in dbNSFP. If there are multiple scores, only the most damaging (largest) rankscore is presented. "</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">PROVEAN_pred</dd><dd class="col-sm-6">"If PROVEANori &lt;= -2.5 (rankscore&gt;=0.59) the corresponding NS is predicted as ""D(amaging)""; otherwise it is predicted as ""N(eutral)""."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">PROVEAN_score</dd><dd class="col-sm-6">PROVEAN score (PROVEANori). Scores range from -14 to 14. The smaller the score the more likely the SNP has damaging effect.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">18. target_gene</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">PROVEAN_converted_rankscore</dd><dd class="col-sm-6">"PROVEANori were first converted to PROVEANnew=1-(PROVEANori+14)/28, then ranked among all PROVEANnew scores in dbNSFP. The rankscore is the ratio of the rank the PROVEANnew score over the total number of PROVEANnew scores in dbNSFP. If there are multiple scores, only the most damaging (largest) rankscore is presented. "</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">PROVEAN_pred</dd><dd class="col-sm-6">"If PROVEANori &lt;= -2.5 (rankscore&gt;=0.59) the corresponding NS is predicted as ""D(amaging)""; otherwise it is predicted as ""N(eutral)""."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">PROVEAN_score</dd><dd class="col-sm-6">PROVEAN score (PROVEANori). Scores range from -14 to 14. The smaller the score the more likely the SNP has damaging effect.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">19. clinvar</dt>
          <dd class="col-sm-8">DNA variants related to human diseases/phenotypes<p class="text-secondary">Disease-related variants (SNVs and indels)</p></dd>
          <dd class="col-sm-1"><a href="https://www.ncbi.nlm.nih.gov/clinvar/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">clinvar_clnsig</dd><dd class="col-sm-6">"clinical significance by clinvar 2 - Benign, 3 - Likely benign, 4 - Likely pathogenic, 5 - Pathogenic, 6 - drug response, 7 - histocompatibility. A negative score means the score is for the ref allele"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">clinvar_golden_stars</dd><dd class="col-sm-6">"ClinVar Review Status summary 0 - no assertion criteria provided, 1 - criteria provided, single submitter, 2 - criteria provided, multiple submitters, no conflicts, 3 - reviewed by expert panel, 4 - practice guideline"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">clinvar_rs</dd><dd class="col-sm-6">rs number by clinvar</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">clinvar_trait</dd><dd class="col-sm-6">the trait/disease the clinvar_clnsig referring to</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">20. VEST3</dt>
          <dd class="col-sm-8">Variant Effect Scoring Tool - VEST3 scores on dbNSFP<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"><a href="https://karchinlab.org/apps/appVest.html"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEST3_rankscore</dd><dd class="col-sm-6">VEST3 scores were ranked among all VEST3 scores in dbNSFP. The rankscore is the ratio of the rank of the score over the total number of VEST3 scores in dbNSFP. The scores range from 0 to 1.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEST3_score</dd><dd class="col-sm-6">"VEST 3.0 score. Score ranges from 0 to 1. The larger the score the more likely the mutation may cause functional change. In case there are multiple scores for the same variant, the largest score (most damaging) is presented."</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">21. funseq_noncoding_score</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEST3_rankscore</dd><dd class="col-sm-6">VEST3 scores were ranked among all VEST3 scores in dbNSFP. The rankscore is the ratio of the rank of the score over the total number of VEST3 scores in dbNSFP. The scores range from 0 to 1.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEST3_score</dd><dd class="col-sm-6">"VEST 3.0 score. Score ranges from 0 to 1. The larger the score the more likely the mutation may cause functional change. In case there are multiple scores for the same variant, the largest score (most damaging) is presented."</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">22. FATHMM</dt>
          <dd class="col-sm-8">dbNSFP annotation by FATHMM. Functional Analysis through Hidden Markov Models -- A genome-wide deleteriousness prediction score for SNVs based on 10 feature groups<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"><a href="http://fathmm.biocompute.org.uk/fathmmMKL.htm"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">FATHMM_pred</dd><dd class="col-sm-6">"If a FATHMMori score is &lt;=-1.5 (or rankscore &gt;=0.81415) the corresponding NS is predicted as ""D(AMAGING)""; otherwise it is predicted as ""T(OLERATED)"". Multiple predictions separated by "";"""</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">FATHMM_rankscore</dd><dd class="col-sm-6">"FATHMMnew=-FATHMMori were ranked among all FATHMMnew scores in dbNSFP. The rankscore is the ratio of the rank of the score over the total number of FATHMMnew scores in dbNSFP. If there are multiple scores, only the most damaging (smallest) rankscore is presented. The scores range from 0 to 1."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">FATHMM_score</dd><dd class="col-sm-6">"FATHMM default score (weighted for human inherited-disease mutations with Disease Ontology) (FATHMMori). Scores range from -18.09 to 11.0. The smaller the score the more likely the SNP has damaging effect. Multiple scores separated by "";"" Please refer to Shihab et al. (2013) Human Mutation 34(1):57-65 for details."</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">23. COSMIC</dt>
          <dd class="col-sm-8">Catalogue Of Somatic Mutations In Cancer--Somatic coding variants discovered in cancer<p class="text-secondary">Disease-related variants (SNVs and indels)</p></dd>
          <dd class="col-sm-1"><a href="https://cancer.sanger.ac.uk/cosmic/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">COSMIC_CNT</dd><dd class="col-sm-6">number of samples having this SNV in the COSMIC database</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">COSMIC_ID</dd><dd class="col-sm-6">ID of the SNV at the COSMIC database</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">24. RegulomeDB</dt>
          <dd class="col-sm-8">A genome-wide categorical functional prediction score for SNVs based on ENCODE annotation<p class="text-secondary">functional prediction scores for non-coding SNVs</p></dd>
          <dd class="col-sm-1"><a href="http://regulomedb.org/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">RegulomeDB_motif</dd><dd class="col-sm-6">motif the SNP resides (from RegulomeDB)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">RegulomeDB_score</dd><dd class="col-sm-6">"categorical score from RegulomeDB. The smaller, the more likely the SNP affects binding"</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">25. MetaLR</dt>
          <dd class="col-sm-8">dbNSFP annotation by MetaLR score. It is a ensembl score approach developed in the K. Wang lab.<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MetaLR_pred</dd><dd class="col-sm-6">"Prediction of our MetaLR based ensemble prediction score,""T(olerated)"" or ""D(amaging)"". The score cutoff between ""D"" and ""T"" is 0.5. The rankscore cutoff between ""D"" and ""T"" is 0.82268."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MetaLR_rankscore</dd><dd class="col-sm-6">MetaLR scores were ranked among all MetaLR scores in dbNSFP. The rankscore is the ratio of the rank of the score over the total number of MetaLR scores in dbNSFP. The scores range from 0 to 1.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MetaLR_score</dd><dd class="col-sm-6">"Our logistic regression (LR) based ensemble prediction score, which incorporated 10 scores (SIFT, PolyPhen-2 HDIV, PolyPhen-2 HVAR, GERP++, MutationTaster, Mutation Assessor, FATHMM, LRT, SiPhy, PhyloP) and the maximum frequency observed in the 1000 genomes populations. Larger value means the SNV is more likely to be damaging. Scores range from 0 to 1."</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">26. FATHMM-MKL</dt>
          <dd class="col-sm-8">A genome-wide deleteriousness prediction score for SNVs based on 10 feature groups<p class="text-secondary">Functional prediction scores for non-coding SNVs</p></dd>
          <dd class="col-sm-1"><a href="http://fathmm.biocompute.org.uk/fathmmMKL.htm"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">fathmm-MKL_coding_group</dd><dd class="col-sm-6">"fathmm-MKL coding group, the feature group used for the coding prediction."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">fathmm-MKL_coding_pred</dd><dd class="col-sm-6">"If a fathmm-MKL_coding_score is &gt;0.5 the corresponding nsSNV is predicted as ""D(AMAGING)""; otherwise it is predicted as ""N(EUTRAL)""."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">fathmm-MKL_coding_rankscore</dd><dd class="col-sm-6">the rank of the fathmm-MKL_coding_score among all fathmm-MKL_coding_scores in genome</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">fathmm-MKL_coding_score</dd><dd class="col-sm-6">"fathmm-MKL coding prediction probability, the larger the number the more likely damaging the threshold separating deleterious prediction and neutral prediction is 0.5."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">fathmm-MKL_non-coding_group</dd><dd class="col-sm-6">"fathmm-MKL non-coding group, the feature group used for the non-coding prediction"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">fathmm-MKL_non-coding_pred</dd><dd class="col-sm-6">"If a fathmm-MKL_non-coding_score is &gt;0.5 the corresponding nsSNV is predicted as ""D(AMAGING)""; otherwise it is predicted as ""N(EUTRAL)""."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">fathmm-MKL_non-coding_rankscore</dd><dd class="col-sm-6">the rank of the fathmm-MKL_non-coding_score among all fathmm-MKL_non-coding_scores in genome</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">fathmm-MKL_non-coding_score</dd><dd class="col-sm-6">"fathmm-MKL non-coding prediction probability, the larger the number the more likely damaging; the threshold separating deleterious prediction and neutral prediction is 0.5."</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">27. REVEL</dt>
          <dd class="col-sm-8">"dbNSFP annotation by REVEL. REVEL is a new ensemble method for predicting the pathogenicity of missense variants based on a combination of scores from 13 individual tools: MutPred, FATHMM v2.3, VEST 3.0, Polyphen-2, SIFT, PROVEAN, MutationAssessor, MutationTaster, LRT, GERP++, SiPhy, phyloP, and phastCons. "<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"><a href="https://labworm.com/tool/revel"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">REVEL_rankscore</dd><dd class="col-sm-6">REVEL scores were ranked among all REVEL scores in dbNSFP. The rankscore is the ratio of the rank of the score over the total number of REVEL scores in dbNSFP.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">REVEL_score</dd><dd class="col-sm-6">REVEL is an ensemble score based on 13 individual scores for predicting the pathogenicity of missense variants.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">28. GERP</dt>
          <dd class="col-sm-8">"Genomic Evolutionary Rate Profiling -- A conservation score measured by ""Rejected Substitutions"""<p class="text-secondary">Conservation scores</p></dd>
          <dd class="col-sm-1"><a href="http://mendel.stanford.edu/SidowLab/downloads/gerp/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GERP_NR</dd><dd class="col-sm-6">GERP++ neutral rate</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GERP_RS</dd><dd class="col-sm-6">"GERP++ RS score, the larger the score, the more conserved the site"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GERP_RS_rankscore</dd><dd class="col-sm-6">the rank of the GERP_RS score among all GERP_RS scores in genome</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">29. Reliability_index</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GERP_NR</dd><dd class="col-sm-6">GERP++ neutral rate</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GERP_RS</dd><dd class="col-sm-6">"GERP++ RS score, the larger the score, the more conserved the site"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">GERP_RS_rankscore</dd><dd class="col-sm-6">the rank of the GERP_RS score among all GERP_RS scores in genome</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">30. ORegAnno</dt>
          <dd class="col-sm-8">Known regulatory elements in human genome<p class="text-secondary">other annotation</p></dd>
          <dd class="col-sm-1"><a href="http://www.oreganno.org/dump/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ORegAnno_PMID</dd><dd class="col-sm-6">the PMID of the paper describing the regulation</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ORegAnno_type</dd><dd class="col-sm-6">the type of regulatory region by ORegAnno</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">31. ALSPAC</dt>
          <dd class="col-sm-8">Whole genome allele frequencies from TWINSUK cohort (The Avon Longitudinal Study of Parents and Children (ALSPAC))<p class="text-secondary">Allele frequencies (SNVs and indels)</p></dd>
          <dd class="col-sm-1"><a href="https://www.uk10k.org/studies/cohorts.html"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ALSPAC_AC</dd><dd class="col-sm-6">Alternative allele count in called genotypes in UK10K TWINSUK cohort.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ALSPAC_AF</dd><dd class="col-sm-6">Alternative allele frequency in called genotypes in UK10K TWINSUK cohort.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ALSPAC_AN</dd><dd class="col-sm-6">Total allele count in called genotypes in UK10K TWINSUK cohort.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">32. FANTOM5</dt>
          <dd class="col-sm-8">"Predicted enhancers, enhancer targets, enhancer expression, CAGE peaks including TSS (promoter)"<p class="text-secondary">epigenomics</p></dd>
          <dd class="col-sm-1"><a href="http://fantom.gsc.riken.jp/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">FANTOM5_CAGE_peak_permissive</dd><dd class="col-sm-6">whether the site is within a FANTOM5 phase 1 permissive Cap Analysis of Gene Expression (CAGE) peak permissive set. Y (Yes) or N (No). A CAGE peak generally suggests a promoter region</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">FANTOM5_CAGE_peak_robust</dd><dd class="col-sm-6">whether the site is within a FANTOM5 phase 1+2 robust Cap Analysis of Gene Expression (CAGE) peak robust set. Y (Yes) or N (No). A CAGE peak generally suggests a promoter region</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">FANTOM5_enhancer_differentially_expressed_tissue_cell</dd><dd class="col-sm-6">tissue or cell the enhancer expressed differentially (from FANTOM5 phase 1)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">FANTOM5_enhancer_expressed_tissue_cell</dd><dd class="col-sm-6">tissue or cell the enhancer expressed (from FANTOM5 phase 1)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">FANTOM5_enhancer_permissive</dd><dd class="col-sm-6">whether the site is within a FANTOM5 phase 1+2 predicted permissive enhancer Y (Yes) or N (No)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">FANTOM5_enhancer_robust</dd><dd class="col-sm-6">whether the site is within a FANTOM5 phase 1 predicted robust enhancer Y (Yes) or N (No)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">FANTOM5_enhancer_target</dd><dd class="col-sm-6">"if the site is within a FANTOM5 phase 1 predicted robust enhancer, this is the name of the FANTOM5 predicted target gene of the enhancer."</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">33. MutPred</dt>
          <dd class="col-sm-8">"dbNSFP annotation by MutPred score. MutPred2 is a standalone and web application developed to classify amino acid substitutions as pathogenic or benign in human. In addition, it predicts their impact on over 50 different protein properties and, thus, enables the inference of molecular mechanisms of pathogenicity. "<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"><a href="http://mutpred.mutdb.org/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutPred_AAchange</dd><dd class="col-sm-6">Amino acid change used for MutPred_score calculation.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutPred_Top5features</dd><dd class="col-sm-6">"Top 5 features (molecular mechanisms of disease) as predicted by MutPred with p values. MutPred_score &gt; 0.5 and p &lt; 0.05 are referred to as actionable hypotheses. MutPred_score &gt; 0.75 and p &lt; 0.05 are referred to as confident hypotheses. MutPred_score &gt; 0.75 and p &lt; 0.01 are referred to as very confident hypotheses."");"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutPred_protID</dd><dd class="col-sm-6">UniProt accession or Ensembl transcript ID used for MutPred_score calculation.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutPred_rankscore</dd><dd class="col-sm-6">MutPred scores were ranked among all MutPred scores in dbNSFP. The rankscore is the ratio of the rank of the score over the total number of MutPred scores in dbNSFP.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutPred_score</dd><dd class="col-sm-6">General MutPred score. Scores range from 0 to 1. The larger the score the more likely the SNP has damaging effect.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">34. funseq2</dt>
          <dd class="col-sm-8">A genome-wide deleteriousness prediction score designed for non-coding somatic SNVs<p class="text-secondary">Functional prediction scores for non-coding SNVs</p></dd>
          <dd class="col-sm-1"><a href="http://funseq2.gersteinlab.org/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">funseq2_noncoding_rankscore</dd><dd class="col-sm-6">the rank of the funseq2_noncoding_score among all funseq2_noncoding_scores in genome</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">funseq2_noncoding_score</dd><dd class="col-sm-6">funseq2 noncoding score range 0-5.4 a weighted score designed for damaging prediction of cancer somatic SNPs</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">35. Esembl</dt>
          <dd class="col-sm-8">"Ensembl Regulatory Build, Regulatory segmentations , TFBS binding predictions, predicted regulatory elements (CTCF Binding Site, Enhancer, Open chromatin, Promoter Flanking Region, TF binding site, Promoter) and their cell type specific activity prediction"<p class="text-secondary">Epigenomics</p></dd>
          <dd class="col-sm-1"><a href="http://uswest.ensembl.org/info/genome/funcgen/regulatory_build.html"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Ensembl_Regulatory_Build_ID</dd><dd class="col-sm-6">Ensembl ID for the regulatory feature</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Ensembl_Regulatory_Build_TFBS</dd><dd class="col-sm-6">TFBS from Ensembl Regulatory Build.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Ensembl_Regulatory_Build_TFBS_matrix</dd><dd class="col-sm-6">TF binding matrix ID</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Ensembl_Regulatory_Build_feature_type</dd><dd class="col-sm-6">"genome segment prediction based on 68 cell types from ENCODE, BLUEPRINT and Roadmap. Predicted states: CTCF_Binding_Site, Enhancer, Open_chromatin, Promoter_Flanking_Region, TF_binding_site, Promoter"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Ensembl_geneid</dd><dd class="col-sm-6">Ensembl gene id</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Ensembl_transcriptid</dd><dd class="col-sm-6">"Ensembl transcript ids (separated by "";"")"</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">36. Interpro_domain</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Ensembl_Regulatory_Build_ID</dd><dd class="col-sm-6">Ensembl ID for the regulatory feature</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Ensembl_Regulatory_Build_TFBS</dd><dd class="col-sm-6">TFBS from Ensembl Regulatory Build.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Ensembl_Regulatory_Build_TFBS_matrix</dd><dd class="col-sm-6">TF binding matrix ID</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Ensembl_Regulatory_Build_feature_type</dd><dd class="col-sm-6">"genome segment prediction based on 68 cell types from ENCODE, BLUEPRINT and Roadmap. Predicted states: CTCF_Binding_Site, Enhancer, Open_chromatin, Promoter_Flanking_Region, TF_binding_site, Promoter"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Ensembl_geneid</dd><dd class="col-sm-6">Ensembl gene id</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Ensembl_transcriptid</dd><dd class="col-sm-6">"Ensembl transcript ids (separated by "";"")"</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">37. VEP</dt>
          <dd class="col-sm-8">Variant Effect Predictor<p class="text-secondary">Pre-computed gene-model based annotation (recommended for large-scale annotation)</p></dd>
          <dd class="col-sm-1"><a href="https://uswest.ensembl.org/info/docs/tools/vep/index.html"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_Amino_Acid_Change</dd><dd class="col-sm-6">amino acid change as to a transcript.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_CANONICAL</dd><dd class="col-sm-6">whether a transcript is canonical.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_CCDS</dd><dd class="col-sm-6">CCDS IDs corresponding to transcripts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_CDS_position</dd><dd class="col-sm-6">CDS position as to a transcript.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_Codon_Change_or_Distance</dd><dd class="col-sm-6">codon change or distance as to a transcript.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_Consequence</dd><dd class="col-sm-6">VEP consequence with Ensembl as gene model.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_Exon_or_Intron_Rank</dd><dd class="col-sm-6">exon (intron) rank as to a transcript.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_Gene_ID</dd><dd class="col-sm-6">Ensembl gene IDs corresponding to transcripts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_Gene_Name</dd><dd class="col-sm-6">Ensembl gene names corresponding to transcripts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_HGVSc</dd><dd class="col-sm-6">variant in format of HGVSc.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_HGVSp</dd><dd class="col-sm-6">variant in format of HGVSp.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_LoF</dd><dd class="col-sm-6">LOF (Loss-of-function) by LOFTEE (Loss-Of-Function Transcript Effect Estimator). </dd><dd class="col-sm-1"><a href="https://github.com/konradjk/loftee"> link </a></dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_LoF_filter</dd><dd class="col-sm-6">LOF filter by LOFTEE.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_LoF_flags</dd><dd class="col-sm-6">LOF flags by LOFTEE.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_LoF_info</dd><dd class="col-sm-6">LOF information by LOFTEE.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_Protein_ID</dd><dd class="col-sm-6">Ensembl protein IDs corresponding to transcripts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_Protein_position</dd><dd class="col-sm-6">protein position as to a transcript.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_STRAND</dd><dd class="col-sm-6">strand of CDS as to a transcript.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_SWISSPROT</dd><dd class="col-sm-6">SWISSPROT IDs corresponding to transcripts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_Transcript_ID</dd><dd class="col-sm-6">Ensembl transcript IDs corresponding to consequences.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_cDNA_position</dd><dd class="col-sm-6">cDNA position as to a transcript.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_ensembl_summary</dd><dd class="col-sm-6">VEP consequence summary with Ensembl as gene model.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_Amino_Acid_Change</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_CANONICAL</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_CDS_position</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_Codon_Change_or_Distance</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_Consequence</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_Exon_or_Intron_Rank</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_Gene_ID</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_Gene_Name</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_HGVSc</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_HGVSp</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_LoF</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_LoF_filter</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_LoF_flags</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_LoF_info</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_Protein_ID</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_Protein_position</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_STRAND</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_Transcript_ID</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_cDNA_position</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">VEP_refseq_summary</dd><dd class="col-sm-6">"same as above, but refseq"</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">38. ANNOVAR</dt>
          <dd class="col-sm-8">Pre-computed ANNOVAR annotations for all alternative SNVs based on human reference genome hg19<p class="text-secondary">Pre-computed gene-model based annotation (recommended for large-scale annotation)</p></dd>
          <dd class="col-sm-1"><a href="http://annovar.openbioinformatics.org/en/latest/user-guide/download/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_Closest_gene(intergenic_only)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_Effect</dd><dd class="col-sm-6">ANNOVAR consequence with RefSeq as gene model.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_Exon_Rank</dd><dd class="col-sm-6">exon number the variant locates (if applicable)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_Gene_ID</dd><dd class="col-sm-6">RefSeq gene IDs corresponding to transcripts</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_HGVSc</dd><dd class="col-sm-6">variant in format of HGVSc</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_HGVSp</dd><dd class="col-sm-6">variant in format of HGVSp.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_Transcript_ID</dd><dd class="col-sm-6">RefSeq transcript IDs corresponding to consequences</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_summary</dd><dd class="col-sm-6">ANNOVAR consequence summary with RefSeq as gene model</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_Closest_gene(intergenic_only)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_Effect</dd><dd class="col-sm-6">ANNOVAR consequence with UCSC knownGene as gene model</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_Exon_Rank</dd><dd class="col-sm-6">exon number the variant locates (if applicable)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_Gene_ID</dd><dd class="col-sm-6">UCSC gene IDs corresponding to transcripts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_HGVSc</dd><dd class="col-sm-6">variant in format of HGVSc.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_HGVSp</dd><dd class="col-sm-6">variant in format of HGVSp.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_Transcript_ID</dd><dd class="col-sm-6">UCSC transcript IDs corresponding to consequences</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_summary</dd><dd class="col-sm-6">ANNOVAR consequence summary with UCSC as gene model.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">39. Motif_breaking</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_Closest_gene(intergenic_only)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_Effect</dd><dd class="col-sm-6">ANNOVAR consequence with RefSeq as gene model.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_Exon_Rank</dd><dd class="col-sm-6">exon number the variant locates (if applicable)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_Gene_ID</dd><dd class="col-sm-6">RefSeq gene IDs corresponding to transcripts</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_HGVSc</dd><dd class="col-sm-6">variant in format of HGVSc</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_HGVSp</dd><dd class="col-sm-6">variant in format of HGVSp.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_Transcript_ID</dd><dd class="col-sm-6">RefSeq transcript IDs corresponding to consequences</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_refseq_summary</dd><dd class="col-sm-6">ANNOVAR consequence summary with RefSeq as gene model</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_Closest_gene(intergenic_only)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_Effect</dd><dd class="col-sm-6">ANNOVAR consequence with UCSC knownGene as gene model</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_Exon_Rank</dd><dd class="col-sm-6">exon number the variant locates (if applicable)</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_Gene_ID</dd><dd class="col-sm-6">UCSC gene IDs corresponding to transcripts.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_HGVSc</dd><dd class="col-sm-6">variant in format of HGVSc.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_HGVSp</dd><dd class="col-sm-6">variant in format of HGVSp.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_Transcript_ID</dd><dd class="col-sm-6">UCSC transcript IDs corresponding to consequences</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ANNOVAR_ucsc_summary</dd><dd class="col-sm-6">ANNOVAR consequence summary with UCSC as gene model.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">40. MutationAssessor</dt>
          <dd class="col-sm-8">dbNSFP annotation by MutationAssessor score<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"><a href="http://mutationassessor.org/r3/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationAssessor_pred</dd><dd class="col-sm-6">"MutationAssessor's functional impact of a variant - predicted functional, i.e. high (""H"") or medium (""M""), or predicted non-functional, i.e. low (""L"") or neutral (""N""). "</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationAssessor_rankscore</dd><dd class="col-sm-6">MAori scores were ranked among all MAori scores in dbNSFP. The rankscore is the ratio of the rank of the score over the total number of MAori scores in dbNSFP. The scores range from 0 to 1.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationAssessor_score</dd><dd class="col-sm-6">MutationAssessor functional impact combined score (MAori). The score ranges from -5.135 to 6.49 in dbNSFP.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">41. fold-degenerate</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationAssessor_pred</dd><dd class="col-sm-6">"MutationAssessor's functional impact of a variant - predicted functional, i.e. high (""H"") or medium (""M""), or predicted non-functional, i.e. low (""L"") or neutral (""N""). "</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationAssessor_rankscore</dd><dd class="col-sm-6">MAori scores were ranked among all MAori scores in dbNSFP. The rankscore is the ratio of the rank of the score over the total number of MAori scores in dbNSFP. The scores range from 0 to 1.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationAssessor_score</dd><dd class="col-sm-6">MutationAssessor functional impact combined score (MAori). The score ranges from -5.135 to 6.49 in dbNSFP.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">42. pos</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationAssessor_pred</dd><dd class="col-sm-6">"MutationAssessor's functional impact of a variant - predicted functional, i.e. high (""H"") or medium (""M""), or predicted non-functional, i.e. low (""L"") or neutral (""N""). "</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationAssessor_rankscore</dd><dd class="col-sm-6">MAori scores were ranked among all MAori scores in dbNSFP. The rankscore is the ratio of the rank of the score over the total number of MAori scores in dbNSFP. The scores range from 0 to 1.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationAssessor_score</dd><dd class="col-sm-6">MutationAssessor functional impact combined score (MAori). The score ranges from -5.135 to 6.49 in dbNSFP.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">43. amino acid position</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">aapos_FATHMM</dd><dd class="col-sm-6">ENSP id (Ensembl protein id) and amino acid positions corresponding to FATHMM scores.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">aapos_SIFT</dd><dd class="col-sm-6">ENSP id and amino acid positions corresponding to SIFT scores.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">44. ESP6500</dt>
          <dd class="col-sm-8">Exome allele frequencies from the Exome Variant Server ESP6500SI-V2 release<p class="text-secondary">Allele frequencies (SNVs and indels)</p></dd>
          <dd class="col-sm-1"><a href="http://evs.gs.washington.edu/EVS/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ESP6500_AA_AC</dd><dd class="col-sm-6">Alternative allele counts in the African American samples of the NHLBI GO Exome Sequencing Project (ESP6500 data set).</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ESP6500_AA_AF</dd><dd class="col-sm-6">Alternative allele frequency in the African American samples of the NHLBI GO Exome Sequencing Project (ESP6500 data set).</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ESP6500_AC</dd><dd class="col-sm-6">Alternative allele counts in the combined African/European American samples of the NHLBI GO Exome Sequencing Project (ESP6500 data set).</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ESP6500_AF</dd><dd class="col-sm-6">Alternative allele frequency in the combined African/European American samples of the NHLBI GO Exome Sequencing Project (ESP6500 data set).</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ESP6500_EA_AC</dd><dd class="col-sm-6">Alternative allele counts in the European American samples of the NHLBI GO Exome Sequencing Project (ESP6500 data set).</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ESP6500_EA_AF</dd><dd class="col-sm-6">Alternative allele frequency in the European American samples of the NHLBI GO Exome Sequencing Project (ESP6500 data set).</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">45. MetaSVN</dt>
          <dd class="col-sm-8">dbNSFP annotation by MetaSVM score. It is a ensembl score approach developed in the K. Wang lab.<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MetaSVM_pred</dd><dd class="col-sm-6">"Prediction of our SVM based ensemble prediction score,""T(olerated)"" or ""D(amaging)"". The score cutoff between ""D"" and ""T"" is 0. The rankscore cutoff between ""D"" and ""T"" is 0.83357. "</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MetaSVM_rankscore</dd><dd class="col-sm-6">MetaSVM scores were ranked among all MetaSVM scores in dbNSFP. The rankscore is the ratio of the rank of the score over the total number of MetaSVM scores in dbNSFP. The scores range from 0 to 1.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MetaSVM_score</dd><dd class="col-sm-6">"Our support vector machine (SVM) based ensemble prediction score, which incorporated 10 scores (SIFT, PolyPhen-2 HDIV, PolyPhen-2 HVAR, GERP++, MutationTaster, Mutation Assessor, FATHMM, LRT, SiPhy, PhyloP) and the maximum frequency observed in the 1000 genomes populations. Larger value means the SNV is more likely to be damaging. Scores range from -2 to 3 in dbNSFP. "</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">46. Uniprot</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_aapos</dd><dd class="col-sm-6">amino acid position as to Uniprot.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_acc</dd><dd class="col-sm-6">Uniprot accession number.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_id</dd><dd class="col-sm-6">Uniprot ID number.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">47. rs_dbSNP150</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_aapos</dd><dd class="col-sm-6">amino acid position as to Uniprot.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_acc</dd><dd class="col-sm-6">Uniprot accession number.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_id</dd><dd class="col-sm-6">Uniprot ID number.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">48. network_hub</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_aapos</dd><dd class="col-sm-6">amino acid position as to Uniprot.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_acc</dd><dd class="col-sm-6">Uniprot accession number.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_id</dd><dd class="col-sm-6">Uniprot ID number.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">49. refcodon</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_aapos</dd><dd class="col-sm-6">amino acid position as to Uniprot.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_acc</dd><dd class="col-sm-6">Uniprot accession number.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_id</dd><dd class="col-sm-6">Uniprot ID number.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">50. ref</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_aapos</dd><dd class="col-sm-6">amino acid position as to Uniprot.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_acc</dd><dd class="col-sm-6">Uniprot accession number.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Uniprot_id</dd><dd class="col-sm-6">Uniprot ID number.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">51. SiPhy</dt>
          <dd class="col-sm-8">SiPhy implements rigorous statistical tests to detect bases under selection from a multiple alignment data. It takes full advantage of deeply sequenced phylogenies to estimate both unlikely substitution patterns as well as slowdowns or accelerations in mutation rates.<p class="text-secondary">conservation scores</p></dd>
          <dd class="col-sm-1"><a href="http://portals.broadinstitute.org/genome_bio/siphy/index.html"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SiPhy_29way_logOdds</dd><dd class="col-sm-6">"SiPhy score based on 29 mammals genomes. The larger the score, the more conserved the site"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SiPhy_29way_logOdds_rankscore</dd><dd class="col-sm-6">the rank of the SiPhy_29way_logOdds score among all SiPhy_29way_logOdds scores in genome</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">52. aapos</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SiPhy_29way_logOdds</dd><dd class="col-sm-6">"SiPhy score based on 29 mammals genomes. The larger the score, the more conserved the site"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SiPhy_29way_logOdds_rankscore</dd><dd class="col-sm-6">the rank of the SiPhy_29way_logOdds score among all SiPhy_29way_logOdds scores in genome</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">53. SLR_test_statistic</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SiPhy_29way_logOdds</dd><dd class="col-sm-6">"SiPhy score based on 29 mammals genomes. The larger the score, the more conserved the site"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">SiPhy_29way_logOdds_rankscore</dd><dd class="col-sm-6">the rank of the SiPhy_29way_logOdds score among all SiPhy_29way_logOdds scores in genome</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">54. ExAC</dt>
          <dd class="col-sm-8">Exome allele frequencies from the Exome Aggregation Consortium release 0.3<p class="text-secondary">allele frequencies (SNVs and indels)</p></dd>
          <dd class="col-sm-1"><a href="http://exac.broadinstitute.org/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_AC</dd><dd class="col-sm-6">"Allele count in total ExAC samples (60,706 samples)"</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_AF</dd><dd class="col-sm-6">Allele frequency in total ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_AFR_AC</dd><dd class="col-sm-6">Adjusted Alt allele counts (DP &gt;= 10 &amp; GQ &gt;= 20) in African &amp; African American ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_AFR_AF</dd><dd class="col-sm-6">Adjusted Alt allele frequency (DP &gt;= 10 &amp; GQ &gt;= 20) in African &amp; African American ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_AMR_AC</dd><dd class="col-sm-6">Adjusted Alt allele counts (DP &gt;= 10 &amp; GQ &gt;= 20) in American ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_AMR_AF</dd><dd class="col-sm-6">Adjusted Alt allele frequency (DP &gt;= 10 &amp; GQ &gt;= 20) in American ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_Adj_AC</dd><dd class="col-sm-6">Adjusted Alt allele counts (DP &gt;= 10 &amp; GQ &gt;= 20) in total ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_Adj_AF</dd><dd class="col-sm-6">Adjusted Alt allele frequency (DP &gt;= 10 &amp; GQ &gt;= 20) in total ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_EAS_AC</dd><dd class="col-sm-6">Adjusted Alt allele counts (DP &gt;= 10 &amp; GQ &gt;= 20) in East Asian ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_EAS_AF</dd><dd class="col-sm-6">Adjusted Alt allele frequency (DP &gt;= 10 &amp; GQ &gt;= 20) in East Asian ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_FIN_AC</dd><dd class="col-sm-6">Adjusted Alt allele counts (DP &gt;= 10 &amp; GQ &gt;= 20) in Finnish ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_FIN_AF</dd><dd class="col-sm-6">Adjusted Alt allele frequency (DP &gt;= 10 &amp; GQ &gt;= 20) in Finnish ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_NFE_AC</dd><dd class="col-sm-6">Adjusted Alt allele counts (DP &gt;= 10 &amp; GQ &gt;= 20) in Non-Finnish European ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_NFE_AF</dd><dd class="col-sm-6">Adjusted Alt allele frequency (DP &gt;= 10 &amp; GQ &gt;= 20) in Non-Finnish European ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_SAS_AC</dd><dd class="col-sm-6">Adjusted Alt allele counts (DP &gt;= 10 &amp; GQ &gt;= 20) in South Asian ExAC samples</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ExAC_SAS_AF</dd><dd class="col-sm-6">Adjusted Alt allele frequency (DP &gt;= 10 &amp; GQ &gt;= 20) in South Asian ExAC samples</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">55. Polyphen2</dt>
          <dd class="col-sm-8">dbNSFP annotation by polyphen-2 score<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"><a href="http://genetics.bwh.harvard.edu/pph2/"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HDIV_pred</dd><dd class="col-sm-6">"Polyphen2 prediction based on HumDiv, ""D"" (""probably damaging"", HDIV score in [0.957,1] or rankscore in [0.52996,0.89917]), ""P"" (""possibly damaging"", HDIV score in [0.453,0.956] or rankscore in [0.34412,0.52842]) and ""B"" (""benign"", HDIV score in [0,0.452] or rankscore in [0.02656,0.34399])."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HDIV_rankscore</dd><dd class="col-sm-6">Polyphen2 HDIV scores were first ranked among all HDIV scores in dbNSFP.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HDIV_score</dd><dd class="col-sm-6">"Polyphen2 score based on HumDiv, i.e. hdiv_prob."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HVAR_pred</dd><dd class="col-sm-6">"Polyphen2 prediction based on HumVar, ""D"" (""probably damaging"", HVAR score in [0.909,1] or rankscore in [0.62955,0.9711]), ""P"" (""possibly damaging"", HVAR in [0.447,0.908] or rankscore in [0.44359,0.62885]) and ""B"" (""benign"", HVAR score in [0,0.446] or rankscore in [0.01281,0.44315]). "</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HVAR_rankscore</dd><dd class="col-sm-6">Polyphen2 HVAR scores were first ranked among all HVAR scores in dbNSFP.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HVAR_score</dd><dd class="col-sm-6">"Polyphen2 score based on HumVar, i.e. hvar_prob."</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">56. ultra_sensitive</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HDIV_pred</dd><dd class="col-sm-6">"Polyphen2 prediction based on HumDiv, ""D"" (""probably damaging"", HDIV score in [0.957,1] or rankscore in [0.52996,0.89917]), ""P"" (""possibly damaging"", HDIV score in [0.453,0.956] or rankscore in [0.34412,0.52842]) and ""B"" (""benign"", HDIV score in [0,0.452] or rankscore in [0.02656,0.34399])."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HDIV_rankscore</dd><dd class="col-sm-6">Polyphen2 HDIV scores were first ranked among all HDIV scores in dbNSFP.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HDIV_score</dd><dd class="col-sm-6">"Polyphen2 score based on HumDiv, i.e. hdiv_prob."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HVAR_pred</dd><dd class="col-sm-6">"Polyphen2 prediction based on HumVar, ""D"" (""probably damaging"", HVAR score in [0.909,1] or rankscore in [0.62955,0.9711]), ""P"" (""possibly damaging"", HVAR in [0.447,0.908] or rankscore in [0.44359,0.62885]) and ""B"" (""benign"", HVAR score in [0,0.446] or rankscore in [0.01281,0.44315]). "</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HVAR_rankscore</dd><dd class="col-sm-6">Polyphen2 HVAR scores were first ranked among all HVAR scores in dbNSFP.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HVAR_score</dd><dd class="col-sm-6">"Polyphen2 score based on HumVar, i.e. hvar_prob."</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">57. codonpos</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HDIV_pred</dd><dd class="col-sm-6">"Polyphen2 prediction based on HumDiv, ""D"" (""probably damaging"", HDIV score in [0.957,1] or rankscore in [0.52996,0.89917]), ""P"" (""possibly damaging"", HDIV score in [0.453,0.956] or rankscore in [0.34412,0.52842]) and ""B"" (""benign"", HDIV score in [0,0.452] or rankscore in [0.02656,0.34399])."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HDIV_rankscore</dd><dd class="col-sm-6">Polyphen2 HDIV scores were first ranked among all HDIV scores in dbNSFP.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HDIV_score</dd><dd class="col-sm-6">"Polyphen2 score based on HumDiv, i.e. hdiv_prob."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HVAR_pred</dd><dd class="col-sm-6">"Polyphen2 prediction based on HumVar, ""D"" (""probably damaging"", HVAR score in [0.909,1] or rankscore in [0.62955,0.9711]), ""P"" (""possibly damaging"", HVAR in [0.447,0.908] or rankscore in [0.44359,0.62885]) and ""B"" (""benign"", HVAR score in [0,0.446] or rankscore in [0.01281,0.44315]). "</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HVAR_rankscore</dd><dd class="col-sm-6">Polyphen2 HVAR scores were first ranked among all HVAR scores in dbNSFP.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">Polyphen2_HVAR_score</dd><dd class="col-sm-6">"Polyphen2 score based on HumVar, i.e. hvar_prob."</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">58. MutationTaster</dt>
          <dd class="col-sm-8">dbNSFP annotation by MutationTaster score<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"><a href="http://www.mutationtaster.org/ChrPos.html"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationTaster_converted_rankscore</dd><dd class="col-sm-6">"The MTori scores were first converted. If the prediction is ""A"" or ""D"" MTnew=MTori; if the prediction is ""N"" or ""P"", MTnew=1-MTori. Then MTnew scores were ranked among all MTnew scores in dbNSFP. The rankscore is the ratio of the rank of the score over the total number of MTnew scores in dbNSFP. The scores range from 0.09067 to 0.80722."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationTaster_pred</dd><dd class="col-sm-6">"MutationTaster prediction, ""A"" (""disease_causing_automatic""), ""D"" (""disease_causing""), ""N"" (""polymorphism"") or ""P"" (""polymorphism_automatic"")."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationTaster_score</dd><dd class="col-sm-6">"MutationTaster p-value (MTori), ranges from 0 to 1."</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">59. TargetScan_context++_score_percentile</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationTaster_converted_rankscore</dd><dd class="col-sm-6">"The MTori scores were first converted. If the prediction is ""A"" or ""D"" MTnew=MTori; if the prediction is ""N"" or ""P"", MTnew=1-MTori. Then MTnew scores were ranked among all MTnew scores in dbNSFP. The rankscore is the ratio of the rank of the score over the total number of MTnew scores in dbNSFP. The scores range from 0.09067 to 0.80722."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationTaster_pred</dd><dd class="col-sm-6">"MutationTaster prediction, ""A"" (""disease_causing_automatic""), ""D"" (""disease_causing""), ""N"" (""polymorphism"") or ""P"" (""polymorphism_automatic"")."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">MutationTaster_score</dd><dd class="col-sm-6">"MutationTaster p-value (MTori), ranges from 0 to 1."</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">60. LRT</dt>
          <dd class="col-sm-8">Likelihood ratio test score of dbNSFP<p class="text-secondary">Functional annotation for missense and splicing SNVs &amp; gene-centric annotation</p></dd>
          <dd class="col-sm-1"><a href="http://www.genetics.wustl.edu/jflab/lrt_query.html"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">LRT_converted_rankscore</dd><dd class="col-sm-6">"LRTori scores were first converted as LRTnew=1-LRTori*0.5 if Omega&lt;1, or LRTnew=LRTori*0.5 if Omega&gt;=1. Then LRTnew scores were ranked among all LRTnew scores in dbNSFP. The rankscore is the ratio of the rank over the total number of the scores in dbNSFP. The scores range from 0.00166 to 0.85682."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">LRT_pred</dd><dd class="col-sm-6">"LRT prediction, D(eleterious), N(eutral) or U(nknown), which is not solely determined by the score."</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">LRT_score</dd><dd class="col-sm-6">"The original LRT two-sided p-value (LRTori), ranges from 0 to 1."</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">61. TWINSUC</dt>
          <dd class="col-sm-8">UK 10K TWINSUK cohort<p class="text-secondary">Allele frequencies (SNVs and indels)</p></dd>
          <dd class="col-sm-1"><a href="https://www.uk10k.org/studies/cohorts.html"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">TWINSUK_AC</dd><dd class="col-sm-6">Alternative allele count in called genotypes in UK10K TWINSUK cohort.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">TWINSUK_AF</dd><dd class="col-sm-6">Alternative allele frequency in called genotypes in UK10K TWINSUK cohort.</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">TWINSUK_AN</dd><dd class="col-sm-6">Total allele count in called genotypes in UK10K TWINSUK cohort.</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">62. ENCODE</dt>
          <dd class="col-sm-8">"DNase clusters (across 125 cell types), uniform TFBS"<p class="text-secondary">Epigenomics</p></dd>
          <dd class="col-sm-1"><a href="http://genome.ucsc.edu/ENCODE/downloads.html"> link </a></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_Dnase_cells</dd><dd class="col-sm-6">number of cell lines supporting a DNase I hypersensitive site</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_Dnase_score</dd><dd class="col-sm-6">the higher the score the stronger the evidence of a DNase I hypersensitive site</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_TFBS</dd><dd class="col-sm-6">name of the transcription factors (separated by ;) if the site is within a TFBS</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_TFBS_cells</dd><dd class="col-sm-6">the cell lines (separated by ;) the TFBS was detected</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_TFBS_score</dd><dd class="col-sm-6">the higher the score the stronger the evidence of the TFBS</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_annotated</dd><dd class="col-sm-6">whether annotated by ENCODE based on funseq-0.1</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">63. #chr</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_Dnase_cells</dd><dd class="col-sm-6">number of cell lines supporting a DNase I hypersensitive site</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_Dnase_score</dd><dd class="col-sm-6">the higher the score the stronger the evidence of a DNase I hypersensitive site</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_TFBS</dd><dd class="col-sm-6">name of the transcription factors (separated by ;) if the site is within a TFBS</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_TFBS_cells</dd><dd class="col-sm-6">the cell lines (separated by ;) the TFBS was detected</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_TFBS_score</dd><dd class="col-sm-6">the higher the score the stronger the evidence of the TFBS</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_annotated</dd><dd class="col-sm-6">whether annotated by ENCODE based on funseq-0.1</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">64. aaalt</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_Dnase_cells</dd><dd class="col-sm-6">number of cell lines supporting a DNase I hypersensitive site</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_Dnase_score</dd><dd class="col-sm-6">the higher the score the stronger the evidence of a DNase I hypersensitive site</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_TFBS</dd><dd class="col-sm-6">name of the transcription factors (separated by ;) if the site is within a TFBS</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_TFBS_cells</dd><dd class="col-sm-6">the cell lines (separated by ;) the TFBS was detected</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_TFBS_score</dd><dd class="col-sm-6">the higher the score the stronger the evidence of the TFBS</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_annotated</dd><dd class="col-sm-6">whether annotated by ENCODE based on funseq-0.1</dd></dl>
        <dl class="row">
          <dt class="col-sm-3">65. cds_strand</dt>
          <dd class="col-sm-8"></dd>
          <dd class="col-sm-1"></dd>
        </dl>
        <dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_Dnase_cells</dd><dd class="col-sm-6">number of cell lines supporting a DNase I hypersensitive site</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_Dnase_score</dd><dd class="col-sm-6">the higher the score the stronger the evidence of a DNase I hypersensitive site</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_TFBS</dd><dd class="col-sm-6">name of the transcription factors (separated by ;) if the site is within a TFBS</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_TFBS_cells</dd><dd class="col-sm-6">the cell lines (separated by ;) the TFBS was detected</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_TFBS_score</dd><dd class="col-sm-6">the higher the score the stronger the evidence of the TFBS</dd></dl><dl class="row sub"><dd class="col-sm-1"><span class="dot"></span></dd><dd class="col-sm-3">ENCODE_annotated</dd><dd class="col-sm-6">whether annotated by ENCODE based on funseq-0.1</dd></dl>
            </div>
        </div>
        </div>
{% endblock %}
